taskrun/CpuTimeResource.py

```diff
@@ -77,7 +77,11 @@ def use(self, task):
 
     # enforce CPU time limit on the ProcessTask
     if isinstance(task, ProcessTask):
-      task.add_prefunc(lambda: (limit_cputime(secs)))
+      if False:  # use later when Python subprocess is smarter
+        task.add_prefunc(lambda: (limit_cputime(secs)))
```

Comment:
I'd prefer a comment instead of dead code
ditto in MemoryResource

Reply:
Done